test/remote/gateways/remote_opp_test.rb

```diff
@@ -18,25 +18,25 @@ def setup
       merchant_transaction_id: "active_merchant_test_complete #{time}",
       address: address,
       description: 'Store Purchase - Books',
-        #      riskWorkflow: true,
-        #      testMode: 'EXTERNAL' # or 'INTERNAL', valid only for test system
+      #      riskWorkflow: true,
```

Comment:
Might make sense to shorten the gap between `#` and the keyword in this comment, too. I have a hunch it at one point lined up with the previous lines, but that clearly stopped being the case even before your patch.